lib/galaxy/datatypes/converters/xtc_to_trr.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="CONVERTER_Xtc_Trr_0" name="XTC to TRR" version="1.0.0" hidden="true">
+    <requirements>
+        <requirement type="package" version="1.9.4">mdtraj</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input1' ./input.xtc &&
+        mdconvert ./input.xtc -o ./output.trr 
+    ]]></command>
+    <inputs>
+        <param format="xtc" name="input1" type="data" label="Choose XTC"/>
+    </inputs>
+    <outputs>
+        <data format="trr" name="output1" from_work_dir="output.trr"/>
```

Comment:
You can combine the converters that convert to the same datatype in a single tool by allowing more input dataset formats.